lib/hooks/use-circuit-title.ts

```diff
@@ -0,0 +1,29 @@
+import { useRunFrameStore } from "lib/components/RunFrameWithApi/store"
+import { guessEntrypoint } from "lib/runner"
+import { useEffect } from "react"
+
+export const useCircuitTitle = () => {
```

Comment:
the way you named it, i would guess it returns the title of the circuit

```suggestion
export const useSyncPageTitle = () => {
```

Reply:
(make sure to title rename as well)